server/bennu-core/src/main/java/org/fenixedu/bennu/core/rest/LogbackResource.java

```diff
@@ -36,18 +36,14 @@
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    public String getAllLoggers() {
+    public JsonObject getAllLoggers() {
         accessControl(Group.managers());
-        if (available) {
-            return Holder.getAllLoggers().toString();
-        } else {
-            return "{ 'loggers': [] }";
-        }
+        return Holder.getAllLoggers();
```

Comment:
This will not work properly, because invoking the method on `Holder` will initialize it, and when the SLF4J backend is not Logback, the initialization for it will fail. As such, this fallback must be done here.